### DISCUSSION TITLE
Upgrading Mastodon 4.0rc to 4.0

### DISCUSSION THREAD

**Discussion by MikeHuntington** [mastodon/mastodon#20699] [Q&A] [answered]
Just curious. I saw the upgrade notes and I know it is targeted towards upgrading from 3.0. I wanted to know if there's something special that needs to be done if upgrading from 4.0rc3 ??

**Comment by vavallee** [comment]
Same question (almost). I am on RC2

**Reply by ghost** [reply]
I see instances on 3.4.6 and not upgraded to 4.0, how is this even possible having insecure instances talking to other instances on different versions?

**Comment by claus** [comment, marked as answer]
I just upgraded from 4.0.0rc3 to 4.0.2, and it _**seems**_ to have worked.
I followed the upgrade notes from [the 4.0.2 release page](https://github.com/mastodon/mastodon/releases/tag/v4.0.2) because I _**assumed**_ that 4.0.0.rcN is to be treated as 4.0.0.
I'm not sure at all about this, though.

**Reply by MikeHuntington** [reply]
Yup upgrading from the 4.0 rc versions to 4.0.x was as simple as following the upgrade steps on the 4.0.2 page.

**Reply by vavallee** [reply]
Thanks for the feedback!
